lib/cleanup.js

```diff
@@ -1,12 +0,0 @@
-function removeLeadingZerosFromStreet(token) {
```

Comment:
The plan with this file is that it would someday be a separate module of OA specific cleanup things. The OA team expressed interest in something like that existing, since it doesn't now.

We don't have to get to that goal today, but tightening the coupling between OA validation/cleanup code and our import pipeline does seem to work against it.